service-mybatis/src/main/java/fi/nls/oskari/mybatis/MyBatisHelper.java

```diff
@@ -15,6 +18,9 @@ public static SqlSessionFactory initMyBatis(DataSource ds, Class<?>... mappers)
     }
 
     public static Configuration getConfig(DataSource ds, Class<?>... mappers) {
+        if (ds == null) {
```

Comment:
Debug?

Reply:
Sure, could be taken out of this and put on another draft PR